### ISSUE TITLE
Is MidiJack support MPE? 

### ISSUE THREAD

**Issue by kevinloustau** [keijiro/MidiJack#34]
I wondered if we can use MPE devices (in my case a roli seaboard)?

**Comment by keijiro** [comment]
No, it doesn't support MPE at the moment. It sounds like a nice-to-have feature but I don't have any device that suports MPE. I'd like to keep this issue opened for a future reminder.
